webapp/crashstats/crashstats/apps.py

```diff
@@ -48,7 +45,7 @@
 
 
 def count_sentry_scrub_error(msg):
-    metrics.incr("sentry_scrub_error", 1)
+    METRICS.incr("webapp.sentry_scrub_error", 1)
```

Comment:
This key changes completely. Before we had `webapp.crashstats.apps.sentry_scrub_error` and now we've got `webapp.sentry_scrub_error` which matches the other services.

After this lands, we'll need to update dashboards.